.github/CONTRIBUTING.md

```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
 
 ## Introduction
```

Comment:
This whole thing is very hard for me to understand, please make it more illiterate-friendly